.env.example

```diff
@@ -82,6 +82,10 @@ export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
 # Default: 404 (200 with permissive traffic policy mode)
 # export BOOKTHIEF_EXPECTED_RESPONSE_CODE=404
 
+# optional: ENABLE_DEBUG_SERVER (true/false)
+# Default: true
+# export ENABLE_DEBUG_SERVER=true
+
```

Comment:
For the demo, do we even want to bother exposing this?

Reply:
I think we should expose it for the demo so people can explore this feature.